docs/user/README.md

```diff
@@ -159,6 +159,45 @@ nodes:
   replicas: 2
 ```
 
+kind also supports multiple control-plane and HA setups using a stacked etcd topology:
+```yaml
+# six node cluster with three control-plane and three worker nodes
+kind: Config
+apiVersion: kind.sigs.k8s.io/v1alpha2
+nodes:
+- role: control-plane
+  replicas: 3
+- role: worker
+  replicas: 3
+- role: external-load-balancer
+```
+
+The `external-load-balancer` node is [HAProxy][haproxy] based and it acts like a load
+balancer in front of the control-plane nodes. This node has to be added explicitly.
```

Comment:
Could we add some clarification on what it means when it says a node has to be added explicitly?
It isn't clear from a first read.